### ISSUE TITLE
French equivalent of `Given` is invariable

### ISSUE THREAD

**Issue by ecattez** [cucumber/common#2071]
### 🤔 What's the problem you're trying to solve?

The equivalent of `Given` in French is invariable. It should be `Etant donné` or `Etant donné que` only.

### ✨ What's your proposed solution?

Remove all existing sentences, other than `Etant donné` and `Etant donné que`:
- Etant donnée
- Etant données
- Etant donnés

### ⛏ Have you considered any alternatives or workarounds?

Let people write French like they want, but that would be direspectful for the language.

----

*This text was originally generated from a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), then edited by hand. [You can modify the template here.](https://github.com/cucumber/.github/edit/main/.github/ISSUE_TEMPLATE/feature_request.md)*

**Comment by mpkorstanje** [comment]
Cucumber JVM generates the annotations based on Gherkin. So I'm moving this to the `common` repo.

The translations can be found in:

https://github.com/cucumber/common/blob/main/gherkin/gherkin-languages.json#L1308

**Comment by aslakhellesoy** [comment]
@ecattes how would you translate the following English step to French:

```gherkin
Given the following cars:
  | year | car                        |
  | 1936 | Bugatti Type 57SC Atlantic |
  | 1948 | Tucker Torpedo             |
```

I would write it like this:

```gherkin
Étant données les voitures suivantes:
  | année  | voiture                    |
  |   1936 | Bugatti Type 57SC Atlantic |
  |   1948 | Tucker Torpedo             |
```

I am not a native French speaker so I might have something to learn about grammar here...

**Comment by ecattez** [comment]
Hello @aslakhellesoy.

It would be `Étant donné les voitures suivantes`. It's a particular thing of French grammar called '' preposition ''. If it was after the car, it would be `Les voitures suivante étant données`.

You could find more info here : https://www.larousse.fr/dictionnaires/francais/%C3%A9tant_donn%C3%A9/31312

**Comment by aslakhellesoy** [comment]
Wow you learn something new every day. I’m all for removing the grammatically incorrect synonyms, but it would require a major release of Gherkin (and anything depending on it).

**Comment by ecattez** [comment]
I agree. Maybe we could depreciate all wrong synonyms before removing them?

**Comment by aslakhellesoy** [comment]
It’s *deprecate* :-). We don’t have a deprecation mechanism for keywords. Maybe we should make one…

**Comment by ecattez** [comment]
If it does not happen often, such a mechanism is not necessary, IMO

**Comment by ciaranmcnulty** [comment]
@stof

**Comment by jbpros** [comment]
> /cc @boucardbruno and @jbpros who added this translation here: [cucumber-attic/gherkin2@v2.5.1...v2.5.2](https://github.com/cucumber-attic/gherkin2/compare/v2.5.1...v2.5.2)

Désolé! 🤓

I must say, now that I see it, I’m embarrassed I let that mistake go through!

**Comment by ecattez** [comment]
It happens. I'm French native and I discovered that rule few days ago. So we were wrong writing scenarios in team 😅

**Comment by ciaranmcnulty** [comment]
I think 'people may write the scenario like this' is more important than grammatical purity though?
